frontend/src/app/components/homescreen/blocks/new-features.component.ts

```diff
@@ -31,6 +31,8 @@ import {DynamicBootstrapper} from "core-app/globals/dynamic-bootstrapper";
 import {I18nService} from "core-app/modules/common/i18n/i18n.service";
 import {DomSanitizer} from "@angular/platform-browser";
 
+export const homescreenNewFeaturesBlockSelector = 'homescreen-new-features-block';
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/3b1eae5a64159f8a044579a25870c772186dbd3f/frontend/src/app/components/homescreen/blocks/new-features.component.ts#L35"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>